docs/package.json

```diff
@@ -8,7 +8,10 @@
     "serve": "docusaurus serve",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
-    "clear": "docusaurus clear"
+    "clear": "docusaurus clear",
+    "lint": "eslint . --ext js,jsx,md,mdx",
+    "prettier:check": "prettier --check .",
+    "prettier:format": "prettier --write ."
```

Comment:
I've been looking around at other examples / best practices.
Perhaps just `check` and `format`?

Reply:
Then again, maybe it's fine.

Reply:
There seems to be different conventions depending where you go.

Reply:
I retract my suggestion.